framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/OpenShiftDeployment.java

```diff
@@ -64,21 +72,26 @@ public void deleteInstances(Instance... instance) {
     @Override
     public void deleteInstances(List<Instance> instances) {
         for (Instance instance : instances) {
-            openShiftController.getClient().pods().inNamespace(namespace).withName(instance.getName()).withGracePeriod(0).delete();
+            Pod pod = util.getPod(instance.getName());
+            util.deletePod(pod, 0);
```

Comment:
Here can be util.deletePod(pod) - grace period is set here to 0. See [OpenShiftUtil](https://github.com/xtf-cz/xtf/blob/master/src/main/java/cz/xtf/openshift/OpenShiftUtil.java#L240)

Reply:
Good catch, thanks. Updated.